metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/UserMetadataService.java

```diff
@@ -93,31 +103,39 @@
      * @return map of uri to data metadata.
      */
     @Nonnull
-    Map<String, ObjectNode> getDataMetadataMap(@Nonnull List<String> uris);
+    default Map<String, ObjectNode> getDataMetadataMap(List<String> uris) {
+        return Collections.emptyMap();
+    }
 
     /**
      * Returns the definition metadata for the given name.
      *
      * @param name name
      * @return definition metadata for the given name
      */
-    Optional<ObjectNode> getDefinitionMetadata(@Nonnull QualifiedName name);
+    default Optional<ObjectNode> getDefinitionMetadata(QualifiedName name) {
+        return Optional.of(JsonNodeFactory.instance.objectNode());
```

Comment:
Why not just return Optional.ofNull instead of returning an actual object? We don't really want anything anyway right?